### DISCUSSION TITLE
Folder headers are not included in the generated curl for each request in the folder

### DISCUSSION THREAD

**Discussion by Billyzou0741326** [Kong/insomnia#8263] [Ideas]
Folder level headers is nice in that it removes duplication on the request level. But, it's not very convenient that `Copy as cURL` and `Generate Code` doesn't include folder level headers.

Is it possible to get folder headers included in the `Copy as cURL` and/or `Generate Code`? If not, are there any plans to introduce it as a feature (or maybe at least have a configuration to toggle this behavior)?

**Comment by subnetmarco** [comment]
This seems to be a bug cc @godfrzero
